test/test_jit.py

```diff
@@ -5043,7 +5043,23 @@ def test_math_ops(self):
         def test_floor():
             return math.floor(1.5)
 
+        def test_fmod(x, y):
+            return math.fmod(x, y)
+
+        def test_modf(x):
+            return math.modf(x)
+
+        def test_remainder(x, y):
+            return math.fmod((y + math.fmod(x, y)), y)
+
+        def test_acos(x):
+            return math.acos(x)
+
         self.checkScript(test_floor, ())
+        self.checkScript(test_fmod, (1.5, 1.))
+        self.checkScript(test_modf, (1.5,))
+        self.checkScript(test_remainder, (1.5, 1.))
```

Comment:
Please add some test cases for negative numbers here, because that's usually the part of the domain where those functions differ!